tests/foreman/cli/test_template.py

```diff
@@ -26,24 +26,17 @@ def test_create_template_1(self):
         @Assert: Template is created
 
         """
-
-        content = gen_string("alpha", 10)
         name = gen_string("alpha", 10)
 
         try:
-            new_obj = make_template(
-                {
-                    'name': name,
-                    'content': content,
-                }
-            )
+            new_obj = make_template({
+                'name': name,
+                'content': gen_string("alpha", 10),
```

Comment:
`s/"/'/g`
`s/, 10//`? (Signature of [`gen_alpha`](https://github.com/omaciel/fauxfactory/blob/master/fauxfactory/__init__.py#L147))